content/gocourse-2019-05-melbourne.slide

```diff
@@ -586,7 +578,7 @@ If `master` has advanced after `feature` branch and PR creation you have two opt
 
 	language:none
 	git checkout master
-	git pull upstream master # or just `git pull` if `master` is set to track `upstream/master`
+	git pull upstream master # or just `git` `pull` if `master` is set to track `upstream/master`
```

Comment:
Revert please - this is already a code block (indented) so the back ticks are rendered ([go present syntax](https://godoc.org/golang.org/x/tools/present)).